packages/pubsub/src/message.js

```diff
@@ -0,0 +1,98 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
If it is a modification to an existing file, it should be 2014 (or whenever the file was published). If it is a brand new file, it should be 2017.